src/client/app/helpers/trusted.snippet.helper.ts

```diff
@@ -18,6 +20,18 @@ class TrustedSnippet {
         }
     }
 
+    reloadTrustedSnippets(): void {
+        try {
+            let trustedSnippets = JSON.parse(window.localStorage.getItem(this._trustedSnippetsKey));
+            for (let snippetId of Object.keys(trustedSnippets)) {
+                if (!storage.snippets.get(snippetId)) {
```

Comment:
This will only load the snippets list once?
Come to think of it, couldn't we just pass in the snippets list to the function outright as a parameter?